README.md

```diff
@@ -36,7 +36,7 @@ developer communities.
 * Self-play mechanism for training agents in adversarial scenarios
 * Train memory-enhanced agents using deep reinforcement learning
 * Easily definable Curriculum Learning and Generalization scenarios
-* Built-in support for Imitation Learning
+* Built-in support for [Imitation Learning](https://github.com/Unity-Technologies/ml-agents/tree/latest_release/docs/Training-Imitation-Learning.md) through Behavioral Cloning or Generative Adversarial Imitation Learning
```

Comment:
why is it linking to latest_release? (similarly for PPO/SAC)

Reply:
I believe this was discussed at a prior meeting, but it's so users who hit the Readme.md page from Github will access the `latest_release` docs and not the `master` docs.

Reply:
Yeah, that was the reason.

Reply:
The issue is that if we rename/move these files then it will result in broken links for older branches. Not a major issue, but calling it out.